temporal/api/cloud/identity/v1/message.proto

```diff
@@ -12,10 +12,12 @@ option csharp_namespace = "Temporalio.Api.Cloud.Identity.V1";
 import "google/protobuf/timestamp.proto";
 
 message AccountAccess {
-    // The role on the account, should be one of [admin, developer, read]
+    // The role on the account, should be one of [admin, developer, read, owner, financeadmin]
```

Comment:
nit: does `owner` have higher precedence than `admin`? may be worth listing in descending order.